contact.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+    <link href="https://fonts.cdnfonts.com/css/jsmath-cmbx10" rel="stylesheet">
+    <title>Home - Ma page avec Bootstrap</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+    <nav class="navbar font navbar-expand-lg navbar-light bg-light w-100">
+        <div class="container-fluid">
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown"
+                aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <ul class="navbar-nav justify-content-center align-items-center w-100">
+                    <li class="nav-item">
+                        <a class="nav-link fs-4" href="index.html">Antoine<br>Bergeron</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link fs-4" aria-current="page" href="apropos.html">À Propos</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link fs-4" href="mesprojets.html">Mes projets</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link fs-4 active fw-bolder" href="contact.html">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+
+
+
+    <div class="bar-noir"></div>
+
+
+    <div class="container mt-3">
+        <div class="row">
+            <div class="d-flex align-items-center">
+                <img src="/assets/moi.jpg" class="p-pic" alt="photo de profil">
+                <div>
+                    <h2 class="fw-bolder padding-h2">27ans, future développeur et chef de profession</h2>
+                    <div class="bar-noir"></div>
+
+                </div>
+            </div>
+
+
+                    <h2 class="fw-bolder padding-h2">Ville de Québec</h2>
+                    <div class="bar-noir"></div>
+
+                    <img class="map" src="/assets/map.png" alt="Carte du Québec">
```

Comment:
The image is not opening. You should have used here the Google Maps APIm not an image.

Reply:
You should use "assets/map.png" instead to use "/assets/map.png". On the server and Unix systems will not recognize the first "/". This problem is present in all your pages.